tokens/src/components/Form/other.json

```diff
@@ -0,0 +1,116 @@
+{
+  "other": {
+    "form": {
+      "control": {
+        "cursor": { "value": "null", "source": "$custom-control-cursor" },
+        "range": {
+          "track": {
+            "cursor": { "value": "pointer", "source": "$custom-range-track-cursor" }
```

Comment:
I think it would be better to have `"range-track-cursor"` as a single key instead of this nesting (since there are no more variables here related to `track` or `cursor` keys)

Reply:
Yes, your right